docs/configuration.md

```diff
@@ -0,0 +1,316 @@
+## NodeHealthCheck Custom Resource
+
+The Node Healthcheck operator requires configuration by creating one or more
+NHC custom resources. An example CR for remediating worker nodes with the
+Self Node Remediation operator looks like this:
+
+```yaml
+apiVersion: remediation.medik8s.io/v1alpha1
+kind: NodeHealthCheck
+metadata:
+  name: nhc-snr-worker
+spec:
+  selector:
+    matchExpressions:
+      - key: node-role.kubernetes.io/control-plane
+        operator: DoesNotExist
+      - key: node-role.kubernetes.io/master
+        operator: DoesNotExist
+  remediationTemplate:
+    apiVersion: self-node-remediation.medik8s.io/v1alpha1
+    kind: SelfNodeRemediationTemplate
+    namespace: <SNR namespace>
+    name: self-node-remediation-resource-deletion-template
+  escalatingRemediations:
+    - remediationTemplate:
+        apiVersion: superfast.medik8s.io/v1alpha1
```

Comment:
Nit: I think folks tend to copy paste those examples, I think I'd consider using valid values or making it very clear that those are dummy values.

Reply:
fair point, improved 👍🏼